corehq/apps/sms/migrations/0026_add_messagingsubevent_case_id_index.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.8 on 2017-10-13 10:50
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('sms', '0025_add_custom_metadata'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='StartEnterpriseBackend',
```

Comment:
is this supposed to be here?

Reply:
yes, see https://github.com/dimagi/commcare-hq/pull/18197#discussion_r144570091